frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -321,6 +325,40 @@ export const WhatsNewMenu = (props: Props) => {
     entries.push(trackerRemoval);
   }
 
+  const phoneAnnouncement: WhatsNewEntry = {
+    title: l10n.getString("whatsnew-feature-phone-header"),
+    snippet: l10n.getString("whatsnew-feature-phone-snippet"),
+    content: (
+      <WhatsNewContent
+        description={l10n.getString("whatsnew-feature-phone-description")}
+        heading={l10n.getString("whatsnew-feature-phone-header")}
+        image={PhoneMaskingHero.src}
+        cta={{
+          type: "upgrade",
+          content: l10n.getString("whatsnew-feature-phone-upgrade-cta"),
+          onClick: () => trackPurchaseStart(),
```

Comment:
I'm wondering if we're doing this elsewhere for phones as well, but `trackPurchaseStart` sets a cookie `clicked-purchase`, so that we can [send an event to GA](https://github.com/mozilla/fx-private-relay/blob/1c5ee036cef7a6b702f1e57dacf28797d66fdc64/frontend/src/pages/accounts/profile.page.tsx#L70-L80) when someone completes the purchase of Premium.

However, that checks for `profile.has_premium`, so I think (?) it won't necessarily fire for phones, and if it does, it doesn't distinguish between regular Premium purchases and phones purchases. So maybe we should rename that `trackPremiumPurchaseStart` and add a new `trackPhonePurchaseStart` that works in a similar way, but for phones?

(Or remove this line altogether and create a separate ticket for that work - I don't know how important tracking these is.)

Reply:
I decided to just remove it for now and create a separate ticket for that work as you've mentioned.